Makefile

```diff
@@ -0,0 +1,18 @@
+.DEFAULT_GOAL := all
+.PRECIOUS: build/flattened/%.sol
+
+CONTRACTS=BRDCrowdsaleAuthorizer BRDLockup BRDToken BRDCrowdsale
+JSONS=build/%.json
+SOLCOPTS=--combined-json abi,asm,ast,bin,bin-runtime,clone-bin,compact-format,devdoc,hashes,interface,metadata,opcodes,srcmap,srcmap-runtime,userdoc --pretty-json
+
+build/flattened/%.sol:
+	truffle-flattener contracts/$*.sol > $@
```

Comment:
use `$(npm bin)/truffle-flattener`.
(before commit it, execute `npm install truffle-flattener --save-dev` and commit `package.json`)